bundles/org.openhab.voice.porcupineks/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+	<config-description uri="voice:porcupineks">
+		<parameter-group name="authentication">
+			<label>Pico Voice Credentials</label>
+		</parameter-group>
+		<parameter-group name="spot">
+			<label>Spot</label>
+		</parameter-group>
```

Comment:
Using groups when you have only one param in each group and only a total of 2 params might be not necessary ?